cmd/launch-taskrun/main_test.go

```diff
@@ -273,14 +299,15 @@ func TestCreateTaskRun_Success(t *testing.T) {
 	mockTekton := &mockTektonClient{}
 	zaplog := &zapLogger{l: zaptest.NewLogger(t)}
 
-	service := NewServiceWithDependencies(mockK8s, mockTekton, zaplog, ServiceConfig{})
+	mockCrtlClient := &mockControllerRuntimeClient{}
+	service := NewServiceWithDependencies(mockK8s, mockTekton, mockCrtlClient, zaplog, ServiceConfig{})
 
-	snapshot := &Snapshot{
+	snapshot := &konflux.Snapshot{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      "test-snapshot",
 			Namespace: "test-namespace",
 		},
-		Spec: json.RawMessage(`{"components":[{"name":"test-component","containerImage":"test-image:latest"}]}`),
+		Spec: json.RawMessage(`{"application":"test-app"}`),
```

Comment:
This json could come back perhaps. I think it should still work. :thinking:

Reply:
It worked. Doesn't impact the behavior, but it makes the tests slighted more realistic looking I guess.